crawler.py

```diff
@@ -1,32 +1,43 @@
+import requests
+
 from bs4 import BeautifulSoup
 
-import requests
 
-def parse():
+def crawl():
+    headers = {
+        'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 '
+                      '(KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36'
+    }
+
+    games = []
 
-	HEADERS = {
-		'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.129 Safari/537.36'
```

Comment:
Also, this line was too long (100+ characters).

PEP-8 recommends to fit your code into 80. Often it hard to fit into 80, but you definitely should do your best to fit into 100.